pkg/controller/installation/core_controller.go

```diff
@@ -600,20 +614,45 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 
 	// Write the discovered configuration back to the API. This is essentially a poor-man's defaulting, and
 	// ensures that we don't surprise anyone by changing defaults in a future version of the operator.
+	// Note that we only write the 'base' installation back. We don't want to write the changes from 'overrides', as those should only
+	// be stored in the 'overrides' resource.
 	if err := r.client.Update(ctx, instance); err != nil {
 		r.SetDegraded("Failed to write defaults", err, reqLogger)
 		return reconcile.Result{}, err
 	}
 
+	// update Installation with 'overrides'
+	overrides := operator.Installation{}
+	if err := r.client.Get(ctx, utils.OverridesInstanceKey, &overrides); err != nil {
+		if !apierrors.IsNotFound(err) {
+			reqLogger.Error(err, "An error occurred when querying the 'overrides' Installation resource")
+			return reconcile.Result{}, err
+		}
+		reqLogger.V(5).Info("no 'overrides' installation found")
+	} else {
+		reqLogger.V(5).Info("merging overrides")
+		instance.Spec = overrideInstallationSpec(instance.Spec, overrides.Spec)
+	}
+
+	reqLogger.V(2).Info("Loaded config", "config", instance)
```

Comment:
With this message we'll be logging out twice "Loaded config" even in cases without an overlay.